libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -346,6 +348,8 @@ public void onAvailabilityChecked(OnAvailabilityChecked event) {
         if (event.isError()) {
             // report the error but don't bail yet.
             AppLog.e(T.API, "OnAvailabilityChecked has error: " + event.error.type + " - " + event.error.message);
+            // hide the keyboard to ensure the link to login using the site address is visible
+            ActivityUtils.hideKeyboardForced(mEmailInput);
```

Comment:
If the event is not `isError()` and also `event.type == EMAIL` and `event.isAvailable`, then we enter [this case here](https://github.com/wordpress-mobile/WordPress-Android/pull/7391/files#diff-81a24078c194c1cab83eb26f8fe1bceeR361) , so we should insert these same 2 lines there as well.

```
// hide the keyboard to ensure the link to login using the site address is visible
ActivityUtils.hideKeyboardForced(mEmailInput);
```

Also, what do you think about replacing [this line here ](https://github.com/wordpress-mobile/WordPress-Android/pull/7391/files#diff-81a24078c194c1cab83eb26f8fe1bceeR363) `EditTextUtils.hideSoftInput(mEmailInput.getEditText());`  (marked deprecated) with `ActivityUtils.hideKeyboardForced(mEmailInput);` to keep it consistent?